packages/dd-trace/src/encode/chunk.js

```diff
@@ -22,6 +22,17 @@ class Chunk {
       this.reserve(length + 1)
       this.length += 1
       this.buffer[offset] = length | 0xa0
+    } else if (length < 0x100) {
+      this.reserve(2)
+      this.length += 2
+      this.buffer[offset] = 0xd9
+      this.buffer[offset + 1] = length
+    } else if (length < 0x10000) { // map 32
+      this.reserve(3)
+      this.length += 3
+      this.buffer[offset] = 0xda
+      this.buffer[offset + 1] = length >> 8
+      this.buffer[offset + 2] = length
```

Comment:
this is just to follow https://github.com/darrachequesne/notepack/blob/master/lib/encode.js#L85-L96. String encoding should need slightly less space

Reply:
In my tests it didn't have any significant impact. If anything, the additional code meant it would be slightly slower in some cases. Do you have any benchmark for this?

Reply:
yeah I'll remove it, I don't think it's worth it